VideoAPI/VideoApi.UnitTests/Controllers/Message/MessageControllerTestBase.cs

```diff
@@ -0,0 +1,43 @@
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System.Collections.Generic;
+using Testing.Common.Helper.Builders.Domain;
+using Video.API.Controllers;
+using VideoApi.DAL.Commands.Core;
+using VideoApi.DAL.Queries.Core;
+using VideoApi.Domain;
+using VideoApi.Domain.Enums;
+
+namespace VideoApi.UnitTests.Controllers
+{
+    public class MessageControllerTestBase
```

Comment:
Again, not necessary since the API tests already verify this behaviour. Mocks do not help here

Reply:
removed the testBase.cs